libs/coin-modules/coin-tezos/src/logic/estimateFees.integ.test.ts

```diff
@@ -34,8 +34,8 @@ describe("estimateFees", () => {
     const result = await estimateFees({ account, transaction });
     // Then
     expect(result).toEqual({
-      estimatedFees: BigInt("866"),
-      fees: BigInt("492"),
+      estimatedFees: BigInt("865"),
+      fees: BigInt("491"),
```

Comment:
Was this an expected change ?

Reply:
Well I tested the reveal on prod and on my branch !
And fees are different (less with the new taquito) and the transaction is passing on my side.